blockdevice/util/disks.go

```diff
@@ -0,0 +1,92 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// Disk reresents disk information obtained by reading /sys/block.
+type Disk struct {
+	// Size disk size in bytes.
+	Size uint64
+	// Model from /sys/block/*/device/model.
+	Model string
+	// DeviceName device name (e.g. /dev/sda).
+	DeviceName string
+}
+
+// GetDisks returns list of disks by reading /sys/block.
+func GetDisks() ([]*Disk, error) {
+	disks := []*Disk{}
+
+	sysblock := "/sys/block"
+
+	devices, err := ioutil.ReadDir(sysblock)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to read /sys/block directory")
+	}
+
+	readFile := func(path string) (string, error) {
+		f, e := os.Open(filepath.Join(sysblock, path))
+
+		if e != nil {
+			return "", errors.Wrap(e, "failed to open file")
```

Comment:
we're now using `fmt.Errorf` from Go stdlib to avoid extra dependency

Reply:
fixed